src/main/scala/centaur/api/CentaurCromwellClient.scala

```diff
@@ -36,7 +38,14 @@ object CentaurCromwellClient {
     sendReceiveFutureCompletion(() => cromwellClient.status(workflow.id))
   }
 
-  def isAlive: Boolean = Try(Await.result(cromwellClient.version, CentaurConfig.sendReceiveTimeout)).isSuccess
+  /*
+    Sends a quick ping to the Cromwell query endpoint. The query endpoint is the only one which both hits the
+    database w/o requiring a workflow id and is idempotent. Not using CromwellClient here as it currently does not support query.
```

Comment:
"idempotent" isn't wrong, but actually this endpoint is "safe" in that it doesn't change server state at all.